spec/unit/pdk/validate/puppet/puppet_epp_spec.rb

```diff
@@ -0,0 +1,319 @@
+require 'spec_helper'
+
+describe PDK::Validate::PuppetEPP do
+  let(:module_root) { File.join('path', 'to', 'test', 'module') }
+  let(:tmpdir) { File.join('/', 'tmp', 'puppet-epp-validate') }
+
+  before(:each) do
+    allow(Dir).to receive(:mktmpdir).with('puppet-epp-validate').and_return(tmpdir)
+    allow(FileUtils).to receive(:remove_entry_secure).with(tmpdir)
+  end
+
+  it 'defines the base validator attributes' do
+    expect(described_class).to have_attributes(
+      name:         'puppet-epp',
+      cmd:          'puppet',
+      spinner_text: a_string_matching(%r{puppet EPP syntax}i),
+    )
+  end
+
+  it_behaves_like 'it accepts .epp targets'
+
+  describe '.pattern_ignore' do
```

Comment:
This test can be removed (I assume it's copy-paste from the .pp tests).  Puppet plans can never end in .epp